simulation_parameters/Constants.cs

```diff
@@ -412,6 +412,11 @@ public static class Constants
     public const float TOOLTIP_DEFAULT_DELAY = 1.0f;
 
     public const float TOOLTIP_FADE_SPEED = 0.1f;
+    
+    /// <summary>
+    ///   Increment to ai membrane microbes to make hovering over small cells easier
```

Comment:
I feel like this should be reworded a bit as this isn't really an increment to the AI membrane, it's extra radius added when checking mouse hovered microbes. Also AI should be upper case.